### ISSUE TITLE
passff causes problems with firefox "Session Restore"

### ISSUE THREAD

**Issue by jounathaen** [passff/passff#412]
### General information.

- Versions
  - Operating system: Linux - Fedora 30. Happened on previous versions of Mint as well
  - Browser: Firefox 67 - 64 Bit, happened with older versions as well
  - PassFF: 1.9
  - Host app: 1.2.1

Status line output: `[09:59:33] ls -> (0) no error message`

---

### Actual behaviour

I have "Restore Session" enabled in Firefox. However, when I start the computer and open the browser, I get a window: "Could not restore session" with a button to restore the session. Clicking the button finally restores the session but it is a bit annoying.

This happens for a longer time now and I now tried every addon separately and it seems to be caused by passff.
In other words: **Firefox session restore works fine if I disable the passff addon and always introduces the "Trouble restoring tabs" page when I enable it.**

### Steps to reproduce the behaviour
1. Enable passff addon
2. Reboot
3. Start Firefox

**Comment by tuxor1337** [comment]
Can somebody please confirm this? I don't have the time to look into this.

**Comment by tuxor1337** [comment]
Closing due to inactivity.
